README.md

```diff
@@ -158,6 +158,8 @@ Rails.configure do |config|
 
   config.rails_db_views.functions_path += %w( /some/function/path )
   config.rails_db_views.functions_extension = "*.sql"
+
+  config.rails_db_views.migration_tasks += %w( db:migrate:with_data )
```

Comment:
this an example - additional rake task which will be handled as db migration task. if you use `data-migrate` gem then you'll need to add this to your configuration